src/js/modules/articles/components/ArticleFooter.js

```diff
@@ -72,20 +73,20 @@ const styles = {
 };
 
 const ArticleFooter = ({ classes, article, subscribe }) => {
+  const { volume, issue, section } = article;
   return (
     <Row className={classes.ArticleFooter}>
       <Col xs={12} sm={12} md={8} lg={8}>
         <p className={classes.inPrint}>
           This article appears in print in&nbsp;
-          <a
-            className={classes.subscribe}
-            href="https://issuu.com/stuyspectator/docs"
-            target="_blank"
-          >
-            {`Volume ${toRoman(article.volume)}, Issue ${article.issue}`}
-            {/* TODO: Lookup table for individual volume/issue links */}
-          </a>
-          .
+          {section.permalink.includes('special-issues') ? (
+            <span><Link to={section.permalink}>{section.name}</Link>, {section.description}</span>
```

Comment:
This renders, for Undercurrents: "This article appears in print in  <Link to="...">Undercurrents</Link>, The Spectator's Winter 2018 magazine.